### PR TITLE
wire up profile password change

### LINKED ISSUE #1149: Hookup My Profile - Password and Authentication to API

### PR DESCRIPTION
**Pull request by na3d**
Closes #1149

### CODE REVIEW

File: src/profile/layouts/AuthenticationPage.js [hunk 1]
```diff
@@ -1,32 +1,42 @@
...
+import { profilePassword } from '~/api/account';
 
-export default class AuthenticationPage extends Component {
+export class AuthenticationPage extends Component {
   constructor() {
     super();
     this.state = {
       password: '',
-      expires: '',
+      expires: '0',
```
Comment (mparke) [inline src/profile/layouts/AuthenticationPage.js]: define this as a number so you don't have to parseInt later?

File: src/api/account.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { fetch } from '~/fetch';
+
+function profileAction(action, body, handleRsp) {
+  return async (dispatch, getState) => {
+    const state = getState();
+    const { token } = state.authentication;
+    const rsp = await fetch(token, `/account/profile/${action}`, { method: 'POST', body });
+    await new Promise(resolve => setTimeout(resolve, 0));
```
Comment (mparke) [inline src/api/account.js]: why?

Comment (na3d) [inline src/api/account.js]: Ah, artifact from a problem I ran into, thank you.

**Comment by mparke** [comment]
issues:

- errors occurring across cards
- do any of the expiry options function except never?

screen shot 2017-02-23 at 11 50 32 am

screen shot 2017-02-23 at 11 50 49 am

**Comment by na3d** [comment]
For expiry, the API doesn't handle DT correctly right now on this endpoint, it's reported.

**Comment by mparke** [comment]
@na3d okay, that's fine, and as for errors being render across cards?

**Pull request merged**